resources/views/community/components/request-list/meta-console-filter.blade.php

```diff
@@ -0,0 +1,22 @@
+<label class="text-xs font-bold" for="filter-by-console-select">Console</label>
+<select id="filter-by-console-id" class="w-full sm:max-w-[240px]" @change="handleConsoleChanged">
+    @if ($selectedConsoleId === null)
+        <option selected>Only supported systems</option>
+    @else 
+        <option value=''>Only supported systems</option>
+    @endif
+
+    @if ($selectedConsoleId == -1)
+        <option selected>All systems</option>
+    @else
+        <option value="-1">All systems</option>
+    @endif
+
+    @foreach ($consoles as $console)
+        @if ($selectedConsoleId == $console['ID'])
+            <option selected>{{ e($console->Name) }}</option>
+        @else
+            <option value="{{ $console['ID'] }}">{{ e($console->Name) }}</option>
```

Comment:
This seems to be doing some sort of escaping:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e07cb572-a59d-4bed-bff1-2396851e00a3)

Reply:
Great catch, it appears this is totally unnecessary and Blade automatically handles what `sanitize_outputs()` does in legacy. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1574/commits/d083e4d665b470fe1c5ac58f6a9d32cde2b9611f.